R/importWQP.R

```diff
@@ -57,8 +57,9 @@ importWQP <- function(obs_url, zip=TRUE, tz=""){
   if(!file.exists(obs_url)){
     
     if(zip){
+      message("zip encoding access still in development")
       temp <- tempfile()
-      # options(timeout = 120)
+      options(timeout = 120)
```

Comment:
Well, now I can't find the reason for 120. For the record, I don't think it's quitting if the download hasn't finished in 120 seconds (plenty of mine take longer than that). I think it's if the service hasn't somehow pinged at least 1 bit back in 120 seconds.